examples/macros.rs

```diff
@@ -1,10 +1,8 @@
 //! This file tests if the macros defined by the hermit-entry crate,
 //! which affect linker .note entries, can actually be used.
 
-#[cfg(feature = "kernel")]
 hermit_entry::define_abi_tag!();
-
-#[cfg(feature = "kernel")]
 hermit_entry::define_entry_version!();
```

Comment:
Why ungate these?

Reply:
because we can gate them in `Cargo.toml`, which makes it clear that the example doesn't make sense without that feature flag.

Reply:
Ah, I see. Can we merge that commit separately?